libs/sql-parser/src/test/java/io/crate/sql/parser/TestStatementBuilder.java

```diff
@@ -852,6 +852,12 @@ public void testCreateTableOptionsMultipleTimesNotAllowed() {
             .hasMessage("line 1:83: mismatched input 'partitioned' expecting {<EOF>, ';'}");
     }
 
+    @Test
+    public void testCreateTableAs() {
```

Comment:
Please move this below: `testCreateTableStmtBuilder()`, and move the `create table ... as` lines in `testCreateTableStmtBuilder` into this method, thx!

Reply:
done. To clarify. You are saying that I should get rid of the testCreateTableAs function, and put the lines in the testCreateTableStmtBuilder, correct?

Reply:
nope, just move it below the `testCreateTableStmtBuilder()`, to be close together,
and the tests inside `testCreateTableStmtBuilder()` which test the `create table AS` move them to your new method: `testCreateTableAs`, thx!

Reply:
Apologies, there are no `CREATE TABLE<tableName> AS` stmts in this method, please ignore that, just add your new method below that one, all fine, and again apologies, I didn't check carefully.

Reply:
Okay sounds good. No worries